### PR TITLE
ContinuousIntegration

### PR DESCRIPTION
**Pull request by MariaJose319**

### CODE REVIEW

**Comment by manuelq12** [comment]
Interesting, i can't see the node.js.yml but I'm able to check the actions were added and run properly

@MariaJose319 Sorry for the inconvenience, can you validate if maybe by doing pull locally I'm able to see the file?

If you do a pull and it doesn't show up let me know here in a comment

File: test/MyFirstApiConsume.test.js [hunk 1]
```diff
@@ -88,4 +88,4 @@ describe('First Api Tests', () => {
         expect(response.data.json).to.eql(body);
     });
```
Comment (manuelq12) [inline test/MyFirstApiConsume.test.js:90]: Delete this empty line

**Comment by manuelq12** [comment]
@MariaJose319 I see why it isn't there, you added it to the main branch already
Remember as we are expecting these changes, they shouldn't be merged directly to the main branch, the must go through a reviewing process

**Comment by manuelq12** [comment]
Fellow reviewers, just in case
https://github.com/MariaJose319/workshop-api-testing-js-QAbootcamp/blob/main/.github/workflows/node.js.yml

**Review by manuelq12 — APPROVED** [review]

**Comment by MariaJose319** [comment]
> @MariaJose319 I see why it isn't there, you added it to the main branch already Remember as we are expecting these changes, they shouldn't be merged directly to the main branch, the must go through a reviewing process

@manuelq12. Thank you, I will take into account your recommendations.

**Pull request merged**